test/extended/util/test.go

```diff
@@ -351,12 +351,6 @@ var (
 			`should check if Kubernetes master services is included in cluster-info`, // Don't run kube-dns
 			`DNS configMap`, // this tests dns federation configuration via configmap, which we don't support yet
 
-			// vSphere tests can be skipped generally
-			`vsphere`,
-			`Cinder`, // requires an OpenStack cluster
-			// See the CanSupport implementation in upstream to determine wether these work.
-			`Ceph RBD`,                              // Works if ceph-common Binary installed (but we can't guarantee this on all clusters).
```

Comment:
Note to reviewers, this **does not** enable ceph tests! This rule does not match any test, ceph tests have something like `[Driver: rbd][Feature: Volumes]`.